jobs/models.py

```diff
@@ -89,13 +89,18 @@ class Meetup(models.Model):
     description = models.TextField()
     is_recurring = models.BooleanField(
         default=False,
-        help_text="Is your meetup recurring?"
     )
     #TODO this field should be required if the is_recurring is True
-    recurrence = models.CharField(max_length=255, blank=True, null=True)
+    recurrence = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        help_text="Provide details of recurrence if applicable."
+    )
     meetup_date = models.DateTimeField(
-        null = True,
-        help_text="This stands for a starting date if the meetup is recurring"
+        null=True,
+        help_text="If this is a recurring meetup/event, please enter a start date.\
+            Date format: YYYY-MM-DD"
```

Comment:
We can use whatever date entry format we want.  YYYY-MM-DD is the default one if you don't specify it in the form field widget.  I don't like it because it's not the usual way we (in Europe at least) enter dates, and because using numbers for months and days is confusing.  2015-01-03 could be the 1st March or the 3rd January, depending on how you read it.  I would prefer to use one like DD-MMM-YYYY, so you enter e.g. 01-Mar-2015.  In the MeetupForm, under class Meta:, use something like:
```
        widgets = {
            'meetup_date': DateTimeInput(format='%d-%b-%Y'),
        }
```

The django docs also have some examples of different date widgets which you could also look at using if you prefer something other than the default text box.
https://docs.djangoproject.com/en/1.7/ref/forms/widgets/#datetimeinput